routes/jwtauth/jwtVerifyAccess.js

```diff
@@ -0,0 +1,43 @@
+import Article from "../../models/post/article";
+import Comment from "../../models/post/comment";
+import Community from "../../models/group/community";
+import Privilege from "../../models/group/privilege";
+
+var _response = {"privileges":null, "verify_error": "ERR_VERIFY_USER_SYNTAX_FAILED"};
+
+const returnResult = (result, error) => {
+    if (result !== undefined) _response.privileges = result;
+    if (error !== undefined) _response.verify_error = error;
+    return _response;
+};
+
+const jwtVerifyAccess = async (userid, type, target) => {
```

Comment:
Codacy found an issue: [Async arrow function has a complexity of 17. Maximum allowed is 4.](https://app.codacy.com/gh/implude/local-community-backend/pullRequest?prid=6225533)